src/config/__jest__/Config.test.ts

```diff
@@ -0,0 +1,99 @@
+import fs from 'fs';
+import Config from '../Config';
+import lodash from 'lodash';
+
+jest.mock('fs');
+describe('Default variables', () => {
```

Comment:
Maybe just `Default config`?

```suggestion
describe('Default config', () => {
```